pepper-apis/parent-pom.xml

```diff
@@ -72,6 +72,12 @@
             <version>1.50.0</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.brsanthu</groupId>
```

Comment:
Google Analytics doesn't provide any official Java library to collect metrics, using this 3rd party library.